alongtheway.txt

```diff
@@ -501,11 +501,11 @@ Read these parables and write down how you feel and what stands out as you read
 
 WEEK SEVEN: MARY
 
-As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a fresh perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
+As I'm writing this we're just coming to the end of the Christmas season and my favorite songs of this season are the songs about Mary and Joseph - exploring the humanity within the Nativity story. Over the last month or two I've thought a lot about Mary and her journey. I wanted to explore her story together and I have been praying that you would come to this story with a new perspective and leave on the other side refreshed and strengthened for whatever God has been calling you to on your own journey with him.
 
 Day 1: overview
 
-During this study we have usually spent the first day of each week doing an overview. Today I'm handing this over to you - I'd encourage you to create your own overview. I'll give you a starting place in Luke but I'd like you to add to it. Create your own outline of the life of Mary, dig into the Word and see what you can find out about her journey.
+During this study we have usually spent the first day of each week doing an overview. Today I'm handing this over to you. I'll give you a starting place in Luke, but I'd like you to add to it. Create your own outline of the life of Mary. Dig into the Word and see what you can find out about her journey.
```

Comment:
Right. Sounds good!

Sent from my iPhone

> On Jan 15, 2016, at 11:14 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> >  
> >  Day 1: overview
> >  
> > -During this study we have usually spent the first day of each week doing an overview. Today I'm handing this over to you - I'd encourage you to create your own overview. I'll give you a starting place in Luke but I'd like you to add to it. Create your own outline of the life of Mary, dig into the Word and see what you can find out about her journey.
> > +During this study we have usually spent the first day of each week doing an overview. Today I'm handing this over to you. I'll give you a starting place in Luke, but I'd like you to add to it. Create your own outline of the life of Mary. Dig into the Word and see what you can find out about her journey.
> remove "i'd encourage you to create your own" since we told them to?
> 
> —
> Reply to this email directly or view it on GitHub.
>